numpy/random/tests/test_random.py

```diff
@@ -467,6 +467,9 @@ def test_shuffle(self):
                      lambda x: [(i, i) for i in x],
                      lambda x: np.asarray([[i, i] for i in x]),
                      lambda x: np.vstack([x, x]).T,
+                     lambda x: (np.asarray([(i, i) for i in x],
+                                           [('', int), ('', int)])
+                                .view(np.recarray)),
```

Comment:
you might want to consider labelling with the corresponding issue you are patching: `gh-11442` ; the test below does that